AABB_tree/include/CGAL/internal/AABB_tree/AABB_drawing_traits.h

```diff
@@ -1,103 +0,0 @@
-// Copyright (c) 2008  INRIA Sophia-Antipolis (France).
-// All rights reserved.
-//
-// This file is part of CGAL (www.cgal.org).
-// You can redistribute it and/or modify it under the terms of the GNU
-// General Public License as published by the Free Software Foundation,
-// either version 3 of the License, or (at your option) any later version.
-//
-// Licensees holding a valid commercial license may use this file in
-// accordance with the commercial license agreement provided with the software.
-//
-// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
-// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
-//
-// $URL$
-// $Id$
-// SPDX-License-Identifier: GPL-3.0+
-//
-//
-// Author(s)     :  Camille Wormser, Pierre Alliez
-
-#ifndef CGAL_AABB_DRAWING_TRAITS_H
-#define CGAL_AABB_DRAWING_TRAITS_H
-
-#include <CGAL/license/AABB_tree.h>
-
-
-#include <CGAL/Bbox_3.h>
-#include <CGAL/gl.h>
```

Comment:
That class `AABB_drawing_traits` was modified by yourself, and no longer uses OpenGL functions.
1. You must not remove this file.
2. You must just remove the `#include <CGAL/gl.h>`.

And try to compile the `cut_plugin`!

Reply:
oops, I only tried the AABB_demo. Didn't think of the cut plugin